drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -0,0 +1,60 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Hud::Household < Hmis::Hud::Base
+  self.table_name = :hmis_households
+  self.primary_key = 'id'
+
+  belongs_to :project, **hmis_relation(:ProjectID, 'Project')
+  has_many :enrollments, **hmis_relation(:HouseholdID, 'Enrollment')
+  has_many :clients, through: :enrollments
+
+  scope :client_matches_search_term, ->(text_search) do
+    joins(:clients).merge(Hmis::Hud::Client.matching_search_term(text_search.to_s))
+  end
+
+  scope :viewable_by, ->(user) do
+    joins(:enrollments).merge(Hmis::Hud::Enrollment.viewable_by(user))
+  end
```

Comment:
You can remove this and include `include ::Hmis::Hud::Concerns::EnrollmentRelated` which defines the same thing